src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const successMessage = `<div class="message">Promise was resolved!</div>`;
+const errorMessage = `<div class="message error-message">Promise was rejected!</div>`;
+
+const successHandler = () => {
+  const body = document.querySelector('body');
+
+  body.insertAdjacentHTML('beforeend', successMessage);
+};
+const errorHandler = () => {
+  const body = document.querySelector('body');
+
+  body.insertAdjacentHTML('beforeend', errorMessage);
```

Comment:
You are querying the `body` element every time the handlers are called. This is not efficient. You should query the `body` element once and store it in a variable outside of the handlers. Then you can use this variable inside your handlers.